tests/php/unit/Configuration/ConfigurationTest.php

```diff
@@ -42,6 +34,7 @@ public function testEmpty() {
 			$configuration->to_array()
 		);
 
+		$this->assertEquals( Filters\applied( 'acf/get_info' ), 1 );
```

Comment:
This only tests the code path for ACF4. Should probably also test the case inside `\Yoast_ACF_Analysis_Configuration->get_acf_version()` that `acf_get_setting()` exists.